core/arch/arm/kernel/generic_entry_a64.S

```diff
@@ -148,6 +148,9 @@ clear_nex_bss:
 	/* Setup SP_EL0 and SP_EL1, SP will be set to SP_EL0 */
 	set_sp
 
+	/* curr_thread needs to be -1 until threads are properly initialized */
+	bl	thread_clr_thread_core_local
```

Comment:
here or after line 155 that enables trapping of exceptions?

Reply:
Exceptions are already trapped, I think it's only asynchronous aborts that are unmasked at that line.

Reply:
Fine.
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`